functions/mod.h

```diff
@@ -3,14 +3,31 @@
 
 #include "svf.h"
 
+class BladeBase;
+
+template<class F, class MAX>
+class ModBase {
+public:
+  void run(BladeBase* blade) {
+    f_.run(blade);
+    max_.run(blade);
+  }
+  int getInteger(int led) {
+    return MOD(f_.getInteger(led), max_.getInteger(led));
+  }
+private:
+  PONUA F f_;
+  PONUA MAX max_;
+};
+
 // Usage: ModF<F, MAX>
 // F: FUNCTION
-// MAX: FUNCTION (not zero) defaults to Int<32768>
+// MAX: FUNCTION (not zero)
```

Comment:
"not zero": Good point, but we should probably check for that just like we do in Divide